### ISSUE TITLE
Trained models not working in Python 3.6+

### ISSUE THREAD

**Issue by yasharmaster** [Vladkryvoruchko/PSPNet-Keras-tensorflow#63]
Loading model files gives error `SystemError: unknown opcode`.
This is the [issue](https://github.com/keras-team/keras/issues/7297) with Keras.
Please provide the models trained on python 3.6.

**Comment by SidShenoy** [comment]
The models can be loaded in the version of python used while creating them. Keras has this issue. Try downgrading python 3.6 to python 3.5, it worked for me.

**Comment by yasharmaster** [comment]
Yeah it worked on my local machine after downgrading the local version of python. But in Google Colab there is no way to downgrade Python 3 version, so it cannot be used there,

**Comment by jmtatsch** [comment]
Just convert the npy weights with your desired keras/python Version. You’ll be fine.

**Comment by CinKKKyo** [comment]
> Yeah it worked on my local machine after downgrading the local version of python. But in Google Colab there is no way to downgrade Python 3 version, so it cannot be used there,

Totally right, so I am going to train it on ADE20K again🤯

**Comment by ktnguyen2** [comment]
@jmtatsch What npy weights? I thought we only used keras weights

**Comment by jmtatsch** [comment]
just search the issues you may find something.
